ui/decredmaterial/line.go

```diff
@@ -36,7 +36,7 @@ func (t *Theme) Line(height, width int) Line {
 
 func (t *Theme) Separator() Line {
 	l := t.Line(1, 0)
-	l.Color = t.Color.Gray1
+	l.Color = t.Color.Background
```

Comment:
Background isn't the correct colour from the mockups. The correct colour is Gray1 with the hex #E6EAED, Background has the hex #444444